test/test_fiber.py

```diff
@@ -0,0 +1,228 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2021-2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+try:
+    import sionna
+except ImportError as e:
+    import sys
+    sys.path.append("../")
+
+import unittest
+import numpy as np
+import tensorflow as tf
+gpus = tf.config.list_physical_devices('GPU')
+print('Number of GPUs available :', len(gpus))
+if gpus:
+    gpu_num = 0 # Number of the GPU to be used
+    try:
+        tf.config.set_visible_devices(gpus[gpu_num], 'GPU')
+        print('Only GPU number', gpu_num, 'used.')
+        tf.config.experimental.set_memory_growth(gpus[gpu_num], True)
+    except RuntimeError as e:
+        print(e)
+from sionna.channel import utils
+from sionna.channel.optical import fiber
+
+
+class TestSSFM(unittest.TestCase):
```

Comment:
For me, all tests fail due to insufficient precision. See the following errors:

FAILED test_fiber.py::TestSSFM::test_batch_processing - AssertionError: 1.1409359796601733e-05 not less than or equal to 1e-07 : incorrect_batch_processing
FAILED test_fiber.py::TestSSFM::test_reference - AssertionError: 1.1409222935614351e-05 not less than or equal to 1e-07 : incorrect_ssfm
FAILED test_fiber.py::TestSSFM::test_reference_graph - AssertionError: 1.1409359796601733e-05 not less than or equal to 1e-07 : incorrect_ssfm_in_graph_mode

Reply:
Here, pytest and unittest (somehow) behaved differently. With unittest everything passed. With pytest I have to explicitly set the dtype to double precision. Fixed this.